sky/serve/service_spec.py

```diff
@@ -64,6 +65,13 @@ def __init__(
                 raise ValueError(
                     f'Unknown load balancing policy: {load_balancing_policy}. '
                     f'Available policies: {list(serve.LB_POLICIES.keys())}')
+
+        if (load_balancer_type is not None and
```

Comment:
as mentioned before, lets check, if using elb:
* lb policy is not specified;
* autoscaling is not enabled (e.g. min_replicas == max_replica).

Reply:
Actually, considering the only policy is rrb and we are about to add least load to our plb, can we support those two policies at least?

Reply:
Mind if I do this in a follow up PR?

Reply:
do this in a follow up PR lgtm, but pls make sure there are proper error for not supported features :))